include/finufftf.h

```diff
@@ -0,0 +1,10 @@
+// minimum definitions needed for interface to FINUFFT library, from C++ or C
```

Comment:
A single header would be like

```
#include <template>
#undef all the things

#define SINGLE
#include <template>
```
This might confuse users who expect to actually see the function declarations in there.  I had something like this, and it is why there is that lost #undef SINGLE missing above...  I think we can do this if that is your preference.